src/ConvertESOMTR/mainModule.f90

```diff
@@ -430,6 +430,7 @@ Subroutine readESOdictionary()
 !'check to make sure the entire file wasn't treated as the data dictionary
 If (lineOfESO /= 'End of Data Dictionary') Then
```

Comment:
If the Do While loop exits at line 418 after reading 'End of Data Dictionary', then where is the next line of the ESO read before line 431?